### PR TITLE
Hex Summit '24 Hands on Lab

### PR DESCRIPTION
**Pull request by aefendic**

### CODE REVIEW

File: site/sfguides/src/hex_titanic/titanic.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
+author: aefendic@hex.tech and chase.romano@snowflake.com
+id: hex
```
Comment (iamontheinet) [inline site/sfguides/src/hex_titanic/titanic.md]: id should be a combination of "End to End Model Development with Hex and Snowpark" with lowercase and "_". for example, "getting_started_with_snowpark_python"

**Review by iamontheinet — CHANGES_REQUESTED** [review]

File: site/sfguides/src/hex_titanic/titanic.md (hunk 1 above)
Comment (aefendic) [reply site/sfguides/src/hex_titanic/titanic.md]: updated, thanks for flagging!

File: site/sfguides/src/hex_titanic/titanic.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
+author: aefendic@hex.tech and chase.romano@snowflake.com
```
Comment (iamontheinet) [inline site/sfguides/src/hex_titanic/titanic.md]: Replace emails with first and last names.

File: site/sfguides/src/hex_titanic/titanic.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+categories: data-science-&-ml,partner-integrations
+environments: web
+status: Published
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Hex, Notebooks, Partner Connect
+
+# End to End Model Development with Hex and Snowpark
+
+<!-- ------------------------ -->
+## Lab Overview
```
Comment (iamontheinet) [inline site/sfguides/src/hex_titanic/titanic.md]: Rename this section to "Overview" and also add "What You Will Build" section.

See https://quickstarts.snowflake.com/guide/getting_started_with_dataengineering_ml_using_snowpark_python/index.html

File: site/sfguides/src/hex_titanic/titanic.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+* How to perform inference using Snowpark and SQL against the registered model
+
+
+### If You Would Like to Create All Assets In The Lab You’ll Need:
+- A [Snowflake](https://signup.Snowflake.com/) Account (if you are using an enterprise account through your organization, it is unlikely that you will have the privileges to use the `ACCOUNTADMIN` role, which is required for this lab).
+
+- A Hex account which will be automatically created via Partner Connect. Please follow the steps below.
+
+<!-- ------------------------ -->
+## Creating a Snowflake account
```
Comment (iamontheinet) [inline site/sfguides/src/hex_titanic/titanic.md]: Remove this step and make it part of the "Prerequisites" section under "Overview" step.

See https://quickstarts.snowflake.com/guide/getting_started_with_dataengineering_ml_using_snowpark_python/index.html

File: site/sfguides/src/hex_titanic/titanic.md [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+Duration: 3
+
+Now that we've completed going through our project, how can we share this project with others? To do this we can head over to the app side of things. At the top of your project, you'll see a button showing if you're in the logic or app view. By switching over to the app side of things, we can start arranging our cells and adding/removing them as we see fit. Check out the [Hex docs](https://learn.hex.tech/docs/build-apps/app-builder) to learn about how you can effectively use the app builder.
+
+A pre-built layout has been designed but feel free to customize and add any additional elements!
+
+Once you are happy with the layout, we can go a head and publish. Publishing is like taking a snapshot of the current state of our app and publishing it as an interactive web app that anyone can see. To publish, hit the green publish button in the top right corner. Once that's done, hit the  share button next to publish, and select `Publish to web`. Whoever you share the link with will be able to see the live version of your app!
+
+
+## Conclusion
```
Comment (iamontheinet) [inline site/sfguides/src/hex_titanic/titanic.md:179]: Make the layout and structure similar to https://quickstarts.snowflake.com/guide/getting_started_with_dataengineering_ml_using_snowpark_python/index.html#0

**Review by iamontheinet — CHANGES_REQUESTED** [review]

**Pull request merged**